### PR TITLE
Update generate endpoint for gen1 template

### PR DESCRIPTION
**Pull request by shudhansu-shekhar**
### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: <shudhansu shekhar> <shudhansu.shekhar.ext@ericsson.com>

### CODE REVIEW

File: service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java [hunk 1]
```diff
@@ -316,10 +322,13 @@ public JsonObject processEvent(String msgProtocol, String msgType, Boolean failI
         JsonObject parsedJson = parsedResponse.getAsJsonObject();
 
         if (parsedJson.has(JSON_ERROR_MESSAGE_FIELD)) {
-            JsonObject eventResponse = new JsonObject();
-            createResponseEntity(HttpStatus.BAD_REQUEST, parsedJson.toString(), JSON_ERROR_STATUS, eventResponse);
-            return eventResponse;
+//            JsonObject eventResponse = new JsonObject();
```
Comment (z-sztrom) [inline service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java:325]: remove unused code

File: service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java [hunk 2]
```diff
@@ -316,10 +322,13 @@ public JsonObject processEvent(String msgProtocol, String msgType, Boolean failI
...
+//            createResponseEntity(HttpStatus.BAD_REQUEST, parsedJson.toString(), JSON_ERROR_STATUS, eventResponse);
+//            return eventResponse;
+            throw new REMGenerateException(response);
         } else {
+           /* JsonObject eventResponse = new JsonObject();
```
Comment (z-sztrom) [inline service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java:330]: remove unused code

**Review by z-sztrom — CHANGES_REQUESTED** [review]

File: service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java [hunk 3]
```diff
@@ -168,46 +169,50 @@ public ResponseEntity<?> generate(final String msgProtocol, final String msgType
...
+                                okToLeaveOutInvalidOptionalFields, element.getAsJsonObject()));
+                        generatedEventResults.add(generatedEvent);
+                        successCount++;
+                    } catch (ProtocolHandlerNotFoundException e) {
+                        // Rethrow the exception. All events in array use the same protocol. If it's handler
+                        // cannot be found for one event, the others will fail, too.
+                        throw e;
+                    } catch (REMGenerateException e) {
+                        // Something went wrong. Add failure description to array of results.
+                      /*  partialSuccess = true;
```
[the remark this answers is not in the document]
Comment (shudhansu-shekhar) [reply service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java:187]: Done

File: service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java (hunk 1 above)
Comment (shudhansu-shekhar) [reply service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java:325]: Done

File: service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java (hunk 2 above)
Comment (shudhansu-shekhar) [reply service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java:330]: Done

**Review by shudhansu-shekhar — COMMENTED** [review]
Remove the unused code, please look

**Review by z-sztrom — APPROVED** [review]

**Pull request merged**